python_modules/dagster/dagster/_cli/definitions.py

```diff
@@ -24,16 +15,7 @@ def definitions_cli():
     """Commands for working with Dagster definitions."""
 
 
-def validate_command_options(f):
-    return apply_click_params(
-        f,
-        generate_workspace_option(),
-        generate_python_file_option(allow_multiple=True),
-        generate_module_name_option(allow_multiple=True),
-    )
-
-
-@validate_command_options
```

Comment:
`validate_command_options` here is just an arbitrary subset of the full set of workspace options. AFAICT there is no reason for this, since we immediately resolve these options to a workspace using the common pathway inside the validate command. Therefore replace with standard workspace options.